### PR TITLE
Gzip redshift deltas

### PR DESCRIPTION
**Pull request by Omicron7**
Minor PR to add gzip support resulted in rewriting the lambda.
It'll now:
- gzip the CSV before sending to S3
- ignore the redshift if csv is empty
- use redis last success key, default to 10 minutes if not found
It should also automatically report all errors through rollbar.

Requires https://github.com/CruGlobal/ecs_config/pull/318

### CODE REVIEW

File: db/redshift.js [hunk 1]
```diff
@@ -4,251 +4,174 @@
...
+      `COPY ${STAGING_PREFIX}${table}
+      FROM 's3://${process.env.REDSHIFT_S3_BUCKET}/${s3Key}'
+      IAM_ROLE '${process.env.REDSHIFT_IAM_ROLE}'
+      CSV BLANKSASNULL EMPTYASNULL TRUNCATECOLUMNS COMPUPDATE OFF STATUPDATE OFF GZIP`
+    const DELETE_QUERY =
+      `DELETE FROM scale_of_belief.${table}
+      USING ${STAGING_PREFIX}${table}
+      WHERE scale_of_belief.${table}.${idColumn} = ${STAGING_PREFIX}${table}.${idColumn}`
+    const INSERT_QUERY =
+      `INSERT INTO scale_of_belief.${table} SELECT * FROM ${STAGING_PREFIX}${table}`
```
Comment (twinge) [inline db/redshift.js]: I realize this comment is unrelated to this PR (other than the fact that this line got refactored): This would be more efficiently done with an APPEND statement:
```
ALTER TABLE scale_of_belief.${table} APPEND FROM ${STAGING_PREFIX}${table}
```

Comment (Omicron7) [reply db/redshift.js]: I believe that would add duplicates since redshift doesn't enforce unique index.

Comment (twinge) [reply db/redshift.js]: It should bring over the exact same data as an `INSERT INTO ... SELECT *` since you don't have a `WHERE` clause on your `SELECT`

Comment (twinge) [reply db/redshift.js]: https://docs.aws.amazon.com/redshift/latest/dg/r_ALTER_TABLE_APPEND.html
